Makefile

```diff
@@ -35,3 +36,11 @@ image-ose-deployer: image-ose-cli
 $(call build-image,ose-recycler,./images/recycler/Dockerfile.rhel,.)
 image-ose-recycler: image-ose-cli
 
+update:
+	hack/update-generated-completions.sh
+.PHONY: update
+
+verify:
```

Comment:
i think make will complain on overriding a target. make a specific target and then verify depend on it. same for update